tools/myst/host/package.c

```diff
@@ -647,6 +649,28 @@ int _exec_package(
         }
     }
 
+    /* Get --main-stack-size */
+    {
+        const char* opt = "--main-stack-size";
+        const char* arg = NULL;
+
+        if ((cli_getopt(&argc, argv, opt, &arg) == 0))
```

Comment:
If the config file does not include main-stack-size entry or has the entry with a specific value, in packaged mode, is commandline option of main-stack-size ignored?

Reply:
The following is true in both package and exec mode: If the config file does not include MainStackSize, the commandline option --main-stack-size will be effective. If config file includes MainStackSize, the commandline option --main-stack-size will be ignored.

Reply:
"If the config file does not include MainStackSize, the commandline option --main-stack-size will be effective" -- this is not right. In packaged mode, the commandline line should be able allowed to change the stacksize. Allowing the untrusted side changing the stacksize might have security implication.

Reply:
Fixed.